include/lsst/meas/extensions/psfex/PsfexPsf.h

```diff
@@ -69,6 +69,10 @@ class PsfexPsf : public lsst::afw::table::io::PersistableFacade<PsfexPsf>,
     bool isPersistable() const noexcept override { return true; }
 
     void write(lsst::afw::table::io::OutputArchiveHandle & handle) const override;
+
+    /// Return the dimensionality of the context data
```

Comment:
Although technically correct, I'm not sure this helps a user know what to do with this; unfortunately, `context` is a psfex-specific term (the only useful docs I could find for it are here: https://psfex.readthedocs.io/en/latest/Working.html ). Maybe this? I'm not sure though.

```suggestion
    /// Return the size of the psfex parameters model.
```

Reply:
How about this?

```Return the number of dependency parameters in the psfex model```

This is closer to the language used in the psfex code and is still understandable to others.

Reply:
Period at the end?

Reply:
```/// Return the number of dependency parameters in the psfex polynomial fit.```